apps/console/src/extensions/configs/models/application.ts

```diff
@@ -153,4 +153,5 @@ export interface ApplicationConfig {
     };
     excludeIdentityClaims: boolean;
     excludeSubjectClaim: boolean;
+    useExtendedRoutes: boolean;
```

Comment:
shall we move this config to common.tsx? WDYT?

Reply:
Addressed with c8aca463e684fb3ea500ebb6f3d333c7a297c53a